python/cuml/cuml/dask/ensemble/__init__.py

```diff
@@ -13,16 +13,5 @@
 # limitations under the License.
 #
 
-import warnings
-
-from cuml.internals.import_utils import has_dask
-
-if has_dask():
-    from cuml.dask.ensemble.randomforestclassifier import (
-        RandomForestClassifier,
-    )
-    from cuml.dask.ensemble.randomforestregressor import RandomForestRegressor
-else:
-    warnings.warn(
```

Comment:
These warnings were completely unnecessary and would never be hit. Any import that tried to import this functionality would have already failed earlier as an error due to missing dask imports.